install_config/aggregate_logging.adoc

```diff
@@ -87,316 +87,314 @@ $ oc project logging
-+
-----
-$ oc secrets new logging-deployer nothing=/dev/null
-----
-
-ifdef::openshift-enterprise[]
-. Create the deployer xref:../admin_guide/service_accounts.adoc#admin-guide-service-accounts[service
-account]:
-+
-====
-----
-$ oc create -f - <<API
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: logging-deployer
-secrets:
-- name: logging-deployer
-API
-----
-====
-endif::openshift-enterprise[]
-ifdef::openshift-origin[]
-. Create the deployer xref:../admin_guide/service_accounts.adoc#admin-guide-service-accounts[service
-account] and custom roles:
-+
-====
 ----
 $ oc new-app logging-deployer-account-template
 ----
-====
-endif::openshift-origin[]
-+
-Then, enable the *logging-deployer* service account to create the objects
-needed for a logging deployment:
-ifdef::openshift-enterprise[]
 +
+[NOTE]
 ====
-----
-$ oc policy add-role-to-user edit --serviceaccount logging-deployer
-----
+If you have deployed logging previously, for example in a different project, then
+it is normal for the cluster roles to fail to be created as they already exist.
 ====
-endif::openshift-enterprise[]
-ifdef::openshift-origin[]
+
+. Enable the deployer service account to create an OAuthClient (normally
+a cluster-admin privilege) for Kibana to use later when authenticating
+against the master.
 +
 ====
 ----
-$ oc policy add-role-to-user edit --serviceaccount logging-deployer
-$ oc policy add-role-to-user daemonset-admin --serviceaccount logging-deployer
 $ oadm policy add-cluster-role-to-user oauth-editor \
        system:serviceaccount:logging:logging-deployer <1>
 ----
-<1> Use the new project you created earlier (e.g., *logging*) when specifying
+<1> Use the project you created earlier (e.g., *logging*) when specifying
 this service account.
 ====
-endif::openshift-origin[]
 
-. Enable the Fluentd service account, which the deployer will create, that
-requires special privileges to operate Fluentd. Add the service account user to
-the security context:
-+
-====
-----
-$ oadm policy add-scc-to-user  \
-    privileged system:serviceaccount:logging:aggregated-logging-fluentd <1>
-----
-<1> Use the new project you created earlier (e.g., *logging*) when specifying
-this service account.
-====
-+
-Give the Fluentd service account permission to read labels from all pods:
+. Enable the Fluentd service account to mount and read system logs by adding
+it to the *privileged* security context, and also enable it to read pod metadata
+by giving it the *cluster-reader* role:
 +
 ====
 ----
+$ oadm policy add-scc-to-user privileged  \
+    system:serviceaccount:logging:aggregated-logging-fluentd <1>
 $ oadm policy add-cluster-role-to-user cluster-reader \
     system:serviceaccount:logging:aggregated-logging-fluentd <1>
 ----
-<1> Use the new project you created earlier (e.g., *logging*) when specifying
+<1> Use the project you created earlier (e.g., *logging*) when specifying
 this service account.
 ====
 
-[[deploying-the-efk-stack]]
-== Deploying the EFK Stack
```

Comment:
add in the anchor above the heading, so:

[[aggregate-logging-specifying-deployer-parameters]]
== Specifying Deployer Parameters